app/src/test/java/org/mozilla/fenix/ext/ImageButtonTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.ext
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import org.mozilla.fenix.TestApplication
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import io.mockk.mockk
+import io.mockk.verify
+import android.widget.ImageButton
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+
+class ImageButtonTest {
+    val mockIb: ImageButton = mockk(relaxed = true)
```

Comment:
This imageButton is created once for the entire class and re-used for every test. This is a problem because this button is stateful and the order the tests are run in can affect whether they pass or not (e.g. in this case, whether or not a certain method was called will persist between tests). You should re-init it for every test using a pattern like this:
```kotlin
class ... {
    private lateinit var imageButton: ImageButton

    @Before
    fun setUp() {
        imageButton = mockk(relaxed =true)
    }
...
```

THat being said, mockk has a short-hand for this:
```kotlin
class ... {
    @MockK(relaxed = true) private lateinit var imageButton: ImageButton

    @Before
    fun setUp() {
        MockKAnnotations.init(this) // this lets the @Mockk annotations work
    }
...
```

This approach isn't simpler when there's only a single mock but it helps when you add more.

Note also that, while it doesn't _really_ matter, it's good to make these things private if they're not intended to be exposed outside of this class.